### ISSUE TITLE
Cant run with java > 1.6.

### ISSUE THREAD

**Issue by mchiareli** [hivewallet/hive-mac#479]
Application won't run with java > 1.6.

It keeps asking for you to install java, even if you have another version installed (e.g java-8).

**Comment by sammy007** [comment]
Yeah, since hive is discontinued I'd be much appreciated for workaround, it seems it's still one and only wallet for mac.

**Comment by mackuba** [comment]
Yeah, it doesn't run with Sun's new Java because it requires a special version of Java from Apple which is somehow better integrated with OSX, which Apple stopped updating after 1.6. Making it run with a different one might be possible, but it's definitely not something that can be done in a few hours. (Especially by someone who hasn't worked much with Java since 2007...)

The Apple Java should install just fine on El Capitan from this package: https://support.apple.com/kb/DL1572?locale=en_US. It will probably stop working on 10.12 though, they've mentioned it in the El Cap release notes...

**Comment by dnivi3** [comment]
@sammy007 and @mchiareli: Electrum runs on OS X and is actively maintained.
